src/components/ui/search-bar/search-bar-renderer.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import CloseIcon from '../../icons/close';
+import Input from '@quantumblack/kedro-ui/lib/components/input';
```

Comment:
This suggests the need to also migrate the `Input` component to our code base as well - @tynandebold have you gotten a chance to check the extent of usage of this component to migrate this as well as part of this ticket?

Reply:
I can see about moving this one too.

Reply:
I've moved it over. It's massively increased the scope of the PR now, but there wasn't much of an alternative.